tencent-cloud/redis/create.go

```diff
@@ -7,24 +7,26 @@ import (
 
 	redis "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/redis/v20180412"
 
+	"github.com/pkg/errors"
 	common "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common"
-	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/errors"
+	tencentErrors "github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/errors"
 	"github.com/tencentcloud/tencentcloud-sdk-go/tencentcloud/common/profile"
 )
 
-type TcloudRedis struct {
-	Region      string `json:"region"`
-	Instance    uint64 `json:"instance"`
-	ZoneId      uint64 `json:"zoneid"`
-	TypeId      uint64 `json:"typeid"`
-	MemSize     uint64 `json:"memsize"`
-	Password    string `json:"password"`
-	ProjectId   int64  `json:"projectid"`
-	Vport       uint64 `json:"vport"`
-	BillingMode int64  `json:"billingmode"`
-	Period      uint64 `json:"billperiod"`
-	VpcId       string `json:"vpcid"`
-	SubnetId    string `json:"subnetid"`
+type TCloudRedis struct {
+	Region       string `json:"region"`
+	InstanceName string `json:"instanceName"`
```

Comment:
TcloudRedis  golang API doesn't have instanceName as input right now :(

Reply:
the API should probably ignore extra data. as long as this doesn't break i am ok with setting this up as an aspiration.

Reply:
We need to fork that repo next week and hook it up.